XamlControlsGallery/ItemTemplates.xaml

```diff
@@ -69,7 +69,7 @@
                     Visibility="{x:Bind BadgeString, Converter={StaticResource nullToVisibilityConverter}}" >
 
                     <Rectangle Fill="{ThemeResource SystemAccentColor}" Opacity=".5" />
-                    <TextBlock Text="{x:Bind BadgeString}" Margin="8,2,8,4" />
+                    <TextBlock Text="{x:Bind BadgeString}" Margin="8,0,8,4" />
```

Comment:
Going from any margin to a 0 margin can be a big jump. Does this impact the text legibility at all?